### PR TITLE
SPCTR-3505 : Resolving store undefined issue widget

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
Resolving widget area issue when drag and drop block inside widget block encountered error.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/inline-notice/settings.js [hunk 1]
```diff
@@ -111,11 +110,8 @@ const Settings = ( props ) => {
 	} = attributes;
 
 	const updateCookieId = ( value ) => {
-		const { getCurrentPostId } = select( 'core/editor' );
-		const post_id = getCurrentPostId().toString();
 		const timestamp = new Date().getTime();
-
-		setAttributes( { c_id: post_id + '-' + timestamp } );
+		setAttributes( { c_id: timestamp } );
```
Comment (AbhijitNage123) [inline src/blocks/inline-notice/settings.js:114]: Is there any reason for removing post_id @rahulvarma722

Comment (rahulvarma722) [reply src/blocks/inline-notice/settings.js:114]: Because this is not required.

Comment (AbhijitNage123) [reply src/blocks/inline-notice/settings.js:114]: Then how we can identify which notice block is used for which page/post in cookies?

Comment (rahulvarma722) [reply src/blocks/inline-notice/settings.js:114]: Put back

**Review by AbhijitNage123 — APPROVED** [review]

File: src/blocks/google-map/wrapper.js [hunk 2]
```diff
@@ -15,8 +15,11 @@ export function GoogleMapsWrapper( ChildComponent ) {
 		const [ scscope = '', scmeta = '' ] = ( attributes?.dynamicContent?.address?.field || '' ).split(
 			'|'
 		);
+
 		// Get the post type of the current post
-		const currentPostType = wp.data.select( 'core/editor' ).getCurrentPostType();
+		const getStore = wp.data.select( 'core/editor' );
```
Comment (adi3890) [inline src/blocks/google-map/wrapper.js:20]: can we import select from @wordpress/data as we have done in [PRO PR](https://github.com/brainstormforce/spectra-pro/blob/500bf5946e53b0461a2bdb9f49c3f56331d57c1e/src/blocks/extensions/dynamic-content/settings/index.js#L2)

**Review by adi3890 — APPROVED** [review]

File: src/blocks/google-map/wrapper.js (hunk 2 above)
Comment (rahulvarma722) [reply src/blocks/google-map/wrapper.js]: Done.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**